ci/svd2rust-regress/src/main.rs

```diff
@@ -93,6 +108,33 @@ fn main() {
         None => &default_svd2rust,
     };
 
+    let default_rustfmt: Option<PathBuf> = if let Some(v) = Command::new("rustup")
+        .args(&["which", "rustfmt"])
+        .output()
+        .ok()
+        .map(|v| v.stdout)
+    {
+        Some(String::from_utf8_lossy(&v).into_owned().trim().into())
+    } else {
+        None
+    }; // FIXME: capture error and assure exit is 0
```

Comment:
@Emilgardis did you want to clean this up before merging?

Reply:
I'm not sure what I want to do about this because if this fails it would simply mean that rustup is missing.
I think a panic if opt.format is true and opt.rustfmt_bin_path would suffice